custom_components/adjustable_bed/beds/leggett_okin.py

```
@@ -0,0 +1,358 @@
+        state = self._motor_state
+        if state.get("head") is True:
+            command += LeggettOkinCommands.MOTOR_HEAD_UP
+        elif state.get("head") is False:
+            command += LeggettOkinCommands.MOTOR_HEAD_DOWN
+        if state.get("feet") is True:
+            command += LeggettOkinCommands.MOTOR_FEET_UP
+        elif state.get("feet") is False:
+            command += LeggettOkinCommands.MOTOR_FEET_DOWN
+        return command
+
+    async def _move_motor(self, motor: str, direction: bool | None) -> None:
```

Ruff (0.14.13)

Boolean-typed positional argument in function definition

(FBT001)